itest/babylon/babylon_e2e_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build e2e_babylon
-// +build e2e_babylon
+//go:build e2e_op
+// +build e2e_op
```

Comment:
if we change the build tag here, it should modify in the Makefile as well

Reply:
Nice find! This is actually a mistake